Apps/W1/ShopifyConnector/app/Products/Tables/ShpfyVariant.Table.al

```diff
@@ -0,0 +1,197 @@
+        field(8; "Display Name"; Text[250])
+        {
+            Caption = 'Display Name';
+            DataClassification = CustomerContent;
+        }
+        field(9; "Inventory Policy"; Enum "Shpfy Inventory Policy")
+        {
+            Caption = 'Inventory Policy';
+            DataClassification = CustomerContent;
+        }
+        field(10; Position; Integer)
+        {
+            Caption = 'Position';
+            DataClassification = CustomerContent;
+        }
+        field(11; Price; Decimal)
+        {
+            Caption = 'Price';
+            DataClassification = CustomerContent;
+        }
+        field(12; SKU; Text[50])
+        {
+            Caption = 'SKU';
+            DataClassification = CustomerContent;
+        }
+        field(13; "Tax Code"; Code[20])
+        {
+            Caption = 'Tax Code';
+            DataClassification = CustomerContent;
+        }
+        field(14; Taxable; Boolean)
+        {
+            Caption = 'Taxable';
+            DataClassification = CustomerContent;
+        }
+        field(15; Title; Text[100])
+        {
+            Caption = 'Title';
+            DataClassification = CustomerContent;
+        }
+        field(16; Weight; Decimal)
+        {
+            Caption = 'Weight';
+            DataClassification = CustomerContent;
+        }
+        field(17; "Option 1 Name"; Text[50])
+        {
+            Caption = 'Option 1 Name';
+            DataClassification = CustomerContent;
+        }
+        field(18; "Option 1 Value"; Text[50])
+        {
+            Caption = 'Option 1 Value';
+            DataClassification = CustomerContent;
+        }
+        field(19; "Option 2 Name"; Text[50])
+        {
+            Caption = 'Option 2 Name';
+            DataClassification = CustomerContent;
+        }
+        field(20; "Option 2 Value"; Text[50])
+        {
+            Caption = 'Option 2 Value';
+            DataClassification = CustomerContent;
+        }
+        field(21; "Option 3 Name"; Text[50])
+        {
+            Caption = 'Option 3 Name';
+            DataClassification = CustomerContent;
+        }
+        field(22; "Option 3 Value"; Text[50])
+        {
+            Caption = 'Option 3 Value';
+            DataClassification = CustomerContent;
+        }
+        field(23; "Unit Cost"; Decimal)
+        {
+            Caption = 'Unit Cost';
+            DataClassification = CustomerContent;
+        }
+        field(24; "Image Id"; BigInteger)
+        {
+            Caption = 'Image Id';
+            DataClassification = CustomerContent;
+        }
+        field(100; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = CustomerContent;
+            TableRelation = "Shpfy Shop";
+        }
+
+        field(101; ItemSystemId; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Item System Id';
+        }
+        field(102; ItemVariantSystemId; Guid)
```

Comment:
please rename field to: "Item Variant SystemId"

Reply:
Updated with commit "Object Renaming"